src/install/log.c

```diff
@@ -197,7 +197,7 @@ int log_meta(int level, const char *file, int line, const char *func, const char
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wformat-nonliteral"
-_noreturn_ static void log_assert(const char *text, const char *file, int line, const char *func, const char *format)
+_noreturn_ static void log_assert(const char *text, const char *file, unsigned int line, const char *func, const char *format)
```

Comment:
Just `unsigned`, please.

Reply:
Shall I use uint32_t?

Reply:
No, why? Just use `unsigned` instead of `unsigned int`. They are synonyms, so use of the longer form is pointless.

Reply:
Well I don't think so. I like to be consistent...
- `unsigned char`
- `unsigned long`
- `unsigned long long`
- `unsigned int`

I don't want to think twice what `C` does with just `unsigned`, when switching programming languages.

Reply:
I understand. I will revert the changes for unsigned.
Much better approach will be using uint32_t, uint64_t, uint8_t and so on from <stdint.h>
For now, I will use unsigned int